### DISCUSSION TITLE
Installing dashboards

### DISCUSSION THREAD

**Discussion by natrlhy** [CarlosCuezva/dashboards-Grafana-Teslamate#19] [Ideas]
I was going to create a PR to update the README.md file to include some initial steps to install and run the `dashboards.sh` script you provide but ran into issues

Here's my fork file https://github.com/natrlhy/dashboards-Grafana-Teslamate/blob/main/README.md#intial-steps-to-login-to-the-container

I select Option1 to create the config file but get:

```
1) Generate config file
2) Download latest release
3) Install/Update dashboards
4) Quit
Please, enter your choice:
1) Generate config file
2) Download latest release
3) Install/Update dashboards
4) Quit
Please, enter your choice: 2
./dashboards.sh: line 166: curl: command not found


./dashboards.sh: line 170: curl: command not found

Downloaded version "" successfully
Press any key to continue
```

Am I not supposed to get into the shell of the grafana container to run all this?

Thanks!

**Comment by CarlosCuezva** [comment]
Hi!

This script is independent of the grafana container, you can use it from any computer within the same network.

**Reply by natrlhy** [reply]
So just provide the configuration info and the URL and location of the dashboard files and it will add it to the TeslaMate instance with the given API key?

**Reply by CarlosCuezva** [reply]
Yep, that's the idea

**Reply by natrlhy** [reply]
I logged into my raspberry pi, ran it and it create the dashboards directory but not accessible to the TeslaMate grafana instance...

```
root@teslamate:~/docker/teslamate# ll
total 4013364
-rw-r--r-- 1 root root      12233 Apr 18 10:00 CHANGELOG.md
-rw-r--r-- 1 root root        230 Apr 18 10:00 config.sh
drwxr-xr-x 2 root root       4096 Apr 18 10:00 dashboards
-rwxr-xr-x 1 root root       4653 Apr 18 10:00 dashboards.sh
-rw-r--r-- 1 root root       1270 Jul 17  2022 docker-compose.yml
drwxr-xr-x 2 root root       4096 Jul 17  2022 import
-rw-r--r-- 1 root root       1070 Apr 18 10:00 LICENSE
-rw-r--r-- 1 root root       6453 Apr 18 10:00 README.md
drwxr-xr-x 2 root root       4096 Apr 18 10:00 screenshots
```
```
root@teslamate:~/docker/teslamate# ll dashboards
total 528
-rw-r--r-- 1 root root 51140 Apr 18 10:00 battery-health.json
-rw-r--r-- 1 root root 38568 Apr 18 10:00 charges.json
-rw-r--r-- 1 root root 76651 Apr 18 10:00 charging-costs-stats.json
-rw-r--r-- 1 root root 24791 Apr 18 10:00 charging-curves.json
-rw-r--r-- 1 root root 77881 Apr 18 10:00 charging-tops.json
-rw-r--r-- 1 root root 44447 Apr 18 10:00 current-charge-view.json
-rw-r--r-- 1 root root 28739 Apr 18 10:00 drives.json
-rw-r--r-- 1 root root 37709 Apr 18 10:00 drive-tops.json
-rw-r--r-- 1 root root 20351 Apr 18 10:00 locations.json
-rw-r--r-- 1 root root 52742 Apr 18 10:00 overview.json
-rw-r--r-- 1 root root 45799 Apr 18 10:00 states.json
-rw-r--r-- 1 root root 12841 Apr 18 10:00 tire-pressure.json
```

**Reply by natrlhy** [reply]
Ok it looks like when I refresh the Grafana page it does see them ;)

![image](https://user-images.githubusercontent.com/19696424/232855523-16fce825-acde-4504-863e-6b2bf7854af7.png)

**Reply by CarlosCuezva** [reply]
perfect  😉

**Comment by natrlhy** [comment]
I saw you mentioned you updated the dashboards. What do we need to do to update them?

**Reply by baylanger** [reply]
> I saw you mentioned you updated the dashboards. What do we need to do to update them?
>
Just run the script, select 2 then 3.
```
2) Download latest release
3) Install/Update dashboards
```

**Reply by natrlhy** [reply]
Thanks!
